### PR TITLE
[Request Review] Corrects the cp calculations for Split-fin rockets.  Fixes #600.

### LINKED ISSUE #600: Calculates Incorrect Cp on split-fin rockets
### Reproduction: 

Open the following in current development branch (`unstable`):
[simple_split_fins.ork.zip](https://github.com/openrocket/openrocket/files/4365711/simple_split_fins.ork.zip)

### Expected 
Same as in 15.03, or the equivalent 4-fin FinSet
Cp.x = 50.5 cm 

### Actual
Cp.x = -32.7 cm

### PR DESCRIPTION
**Pull request by teyrana**
### Inciting Example
[simple_split_fins.ork.zip](https://github.com/openrocket/openrocket/files/4365711/simple_split_fins.ork.zip)

### Relevant Unit Tests:

https://github.com/teyrana/openrocket/blob/2c1daea9d1eea139eb5d55e6ac397e4ec67ef61d/core/test/net/sf/openrocket/aerodynamics/BarrowmanCalculatorTest.java#L114-L226

(prebuilt jar available on request)

Fixes:  #600
May help with: #571, #538

### CODE REVIEW

**Comment by teyrana** [comment]
(edit: rebased to tip of `unstable`)

File: core/src/net/sf/openrocket/util/TestRockets.java [hunk 1]
```diff
@@ -3,7 +3,6 @@
 import java.util.Random;
 
 import net.sf.openrocket.appearance.Appearance;
-import net.sf.openrocket.file.openrocket.OpenRocketSaver;
```
Comment (JoePfeiffer) [inline core/src/net/sf/openrocket/util/TestRockets.java]: This is needed for the dumpRocket() function in PR 603

File: core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java [hunk 2]
```diff
@@ -114,10 +112,12 @@ public void calculateNonaxialForces(FlightConditions conditions, Transformation
 		// Multiple fins with fin-fin interference
 		double cna;
 		double theta = conditions.getTheta();
-		double angle = baseRotation + transform.getXrotation();
```
Comment (JoePfeiffer) [inline core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java]: Does transform.getXrotation completely subsume baseRotation, ie is that member not needed any more?

File: core/src/net/sf/openrocket/util/TestRockets.java [hunk 3]
```diff
@@ -524,6 +524,24 @@ public static final Rocket makeEstesAlphaIII(){
...
+	// This function is used for unit, integration tests, DO NOT CHANGE (without updating tests).
+	public static final void splitRocketFins( BodyTube body, FinSet fins, int finCount){
+		// actually remove the fins
+		body.removeChild(fins);
+
+		FinSet template = fins; 
+		template.setFinCount(1);
+		// and manually add in the equivalent:
+		for(int finNumber=0; finNumber<finCount; ++finNumber){
+			final TrapezoidFinSet singleFin = new TrapezoidFinSet(1, 0.05, 0.03, 0.02, 0.05);
```
Comment (JoePfeiffer) [inline core/src/net/sf/openrocket/util/TestRockets.java]: Could the parameters on these fins come from copying the parameters from the original FinSet?

File: core/src/net/sf/openrocket/util/TestRockets.java (hunk 1 above)
Comment (teyrana) [reply core/src/net/sf/openrocket/util/TestRockets.java]: This change was fixed during rebases.  It's included, now.

File: core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java (hunk 2 above)
Comment (teyrana) [reply core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java:117]: yea, I think so.   But I want further testing to verify that it behaves the appropriate way everywhere else, too.

File: core/src/net/sf/openrocket/util/TestRockets.java (hunk 3 above)
Comment (teyrana) [reply core/src/net/sf/openrocket/util/TestRockets.java:542]: It gave me more control, But you make a good point.  I'll convert it.

Comment (teyrana) [reply core/src/net/sf/openrocket/util/TestRockets.java]: tyg.

**Pull request merged**